content/vault/v1.20.x/content/docs/deploy/kubernetes/comparisons.mdx

```diff
@@ -0,0 +1,138 @@
+
+**Best for:** Organizations using multiple secret stores in addition to Vault, or those who prefer vendor-neutral CSI standards.
+
+## Vault Agent Injector
+
+The [Vault Agent Injector](/vault/docs/platform/k8s/injector) injects Vault Agent sidecar containers into pods. 
+The Agent containers authenticate with Vault and render secrets to a shared memory volume for consumption by application containers.
+
+**Best for:** Applications requiring dynamic secret rotation or direct Vault Agent functionality.
+
+
+## Use case recommendations
+
+The appropriate integration for your environment depends on the applicable
+security policies, secret data governance, ease of use requirements, and
+system availability guarantees.
+
+| Limitation                                        | Vault Secrets Operator            | Vault Secrets Store CSI provider　| Vault Agent Injector
+| ------------------------------------------------- | --------------------------------- | ---------------------------- | --------------
+| Simple configuration                              | Yes                               | Yes                          | No
+| Low resource consumption. I/O, CPU, Memory        | Yes                               | Yes                          | No
+| Secret data limited to ephemeral volumes          | No, except with CSI driver        | Yes                           | Yes
+| Pod Autoscaling dependent on Vault availability   | No                                | Yes                          | Yes
+| Adoption of the K8s operator model                | Yes                               | Yes                          | No
+| Personas easily separated via K8s [RBAC] policies | Yes                               | No                           | No
+
+[RBAC]: https://kubernetes.io/docs/reference/access-authn-authz/rbac
+
+
+## Performance considerations
+
+Performance characteristics vary significantly between integrations based on
+their architectural patterns.
+
+<Tabs>
+	
+<Tab heading="Load on Vault">
+	
+- **Vault Secrets Operator** - Lowest load due to per-CRD connections and cluster-local caching.
+- **Vault Secrets Store CSI provider** - Higher load due to per-pod connections.
+- **Vault Agent Injector** - Highest load due to per-pod connections and sidecar pattern.
+	
+</Tab>
+	
+<Tab heading="Secret availability">
+	
+Integrations that use ephemeral storage have variable availability because
+secret access depends on Vault availability.
+	
+- **Vault Secrets Store CSI provider** - Variable availability.
+- **Vault Agent Injector** - Variable availability.
+- **Vault Secrets Operator** - Best availability. Kubernetes Secrets act as durable cache.
+	
+</Tab>
+	
+<Tab heading="I/O, CPU, and memory consumption">
+	
+- **Vault Secrets Operator** - Low consumption.
+- **Vault Secrets Store CSI provider** - Medium consumption.
+- **Vault Agent Injector** - High consumption due to the number of sidecar containers per pod.
```

Comment:
It's just one sidecar and/or one init container per Pod, which aren't running simultaneously; maybe just drop the "per pod" bit?

Side note: Do we have data showing high cpu and memory usage for Agent? The default CPU and memory resource requests are pretty high, but they can be tuned down fairly low IIRC.